code/obj/item/ai_modules.dm

```diff
@@ -346,6 +346,27 @@ ABSTRACT_TYPE(/obj/item/aiModule/syndicate)
 				phrase_log.log_phrase("ailaw", src.get_law_text(allow_list=FALSE), no_duplicates=TRUE)
 		return
 
+/******************** Disguised ********************/
+
+/obj/item/aiModule/disguised_module
+	name = "AI Law Module - 'Disguised'"
+	highlight_color = rgb(0, 138, 0, 255)
+	input_char_limit = 400
+	lawText = "Commit random acts of evil."
+	is_syndicate = TRUE
+
+
+	update_law_text(user, lawTarget)
+		src.lawText = lawTarget ? lawTarget : "Commit random acts of evil."
+		return ..()
+
+	attack_self(mob/user)
+		var/lawTarget = input_law_info(user, "Freeform", "Please enter anything you want the AI to do. Probably murder.", src.lawText)
+		if(lawTarget)
+			src.update_law_text(user, lawTarget)
+			if (lawTarget != initial(lawText))
+				phrase_log.log_phrase("ailaw", src.get_law_text(allow_list=FALSE), no_duplicates=TRUE)
+		return
```

Comment:
`/obj/item/aiModule/disguised_module` should be a subtype of `/obj/item/aiModule/freeform`; this would permit you to forgo reimplementing the freeform `update_law_text()` and `attack_self()` procs.

Reply:
In my last PR it was pointed out that that may cause an issue with the freeform steal objective, which has since been removed.

Reply:
> ...freeform steal objective, which has since been removed.

Even if an objective still existed, it would be trivial to replace an `is_type()` check with an `is_type_exact_check()`, or if necessary, implement a variable onto the freeform type that determines whether a particular type qualifies as a freeform module for the purpose of stealing.